restart_container_alpha/README.md

```diff
@@ -0,0 +1,10 @@
+# Restart Container (Alpha)
```

Comment:
do we want experimental extensions to have the name `alpha` in them? @landism ? feels odd to me

Reply:
Hmm yeah i might just take off the alpha in the name and keep it unlisted until we've had a few ppl test it out.